components/Slider24.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
```

Comment:
```Slider24``` is not an appropriate name, never use redundant numbers in your variable or component names

Reply:
Error fixed. The error was related to the parts.